k8s/configmaps/apache-tika-configmap.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: apache-tika-config
+data:
+  DD_ENV: "prod"
+  DD_SERVICE: "apache-tika"
+  NODE_OPTIONS: "-r dd-trace/init --max-old-space-size=6000"
+  DD_LOGS_INJECTION: "true"
+  DD_RUNTIME_METRICS_ENABLED: "true"
+  NODE_ENV: "production"
```

Comment:
same here

Reply:
Any reason to not keep this?

Reply:
It's an env var for Node JS which as far as I understand isn't used by apache tika ?

Reply:
(same for NODE_OPTIONS)